production/helm/loki/values.yaml

```diff
@@ -282,6 +285,9 @@ loki:
   # --  Optional querier configuration
   querier: {}
 
+# -- Provides a reloadable runtime configuration file for some specific configuration
+runtimeConfig: {}
```

Comment:
I think `loki.runtimeConfig` is better as you've mentioned.

Reply:
I'm totally agree :)

Reply:
done !